pkg/governance-scripts/contracts/20230518-gauge-adder-migration-v3-to-v4/GaugeAdderMigrationCoordinator.sol

```diff
@@ -31,6 +31,7 @@ contract GaugeAdderMigrationCoordinator is BaseCoordinator {
     ILiquidityGaugeFactory public immutable arbitrumRootGaugeFactory;
     ILiquidityGaugeFactory public immutable optimismRootGaugeFactory;
     ILiquidityGaugeFactory public immutable gnosisRootGaugeFactory;
+    ILiquidityGaugeFactory public immutable zkEvmRootGaugeFactory;
```

Comment:
The contract is PolygonZkEVMRootGaugeFactory (too many "zk" things; wanted to make it extra clear). We could keep it consistent, though I know it's a long name